File: app/src/main/java/com/project/personal/app_bank/models/StatementList.java
package com.project.personal.app_bank.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class StatementList {
    @SerializedName("statementList")
    @Expose
    private List<Statements> statementList;

    public List<Statements> getStatements() {
        return statementList;
    }
}

File: app/src/main/java/com/develop/zupp_bank/domain/models/StatementList.kt
package com.develop.zupp_bank.domain.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

data class StatementList(

    @SerializedName("title")
    @Expose val title: String,

    @SerializedName("desc")
    @Expose val desc: String,

    @SerializedName("date")
    @Expose val date: String,

    @SerializedName("value")
    @Expose val value: Float

)